docs/sphinx/source/whatsnew/v0.5.2.rst

```diff
@@ -14,6 +14,7 @@ Enhancements
 Bug fixes
 ~~~~~~~~~
 * fixed redeclaration of test_simplified_solis_series_elevation (:issue:`387`)
+* removed unused 'pressure' arg from irradiance.liujordan function (:issue:`386`)
```

Comment:
I suggest putting this in the API changes section to give it a little more visibility.